### PR TITLE
cli: show angles also in degrees, and support movement in degrees

### PR DESCRIPTION
**Pull request by pieleric**
Some users find that thinking in degrees is easier than radians... so
let's accomodate them.

### CODE REVIEW

File: src/odemis/cli/main.py [hunk 1]
```diff
@@ -24,21 +24,22 @@
 
 from __future__ import division, print_function
 
-from past.builtins import basestring, unicode
-from builtins import str
 import argparse
+from builtins import str
```
Comment (philipwinklerdelmic) [inline src/odemis/cli/main.py:28]: Do we still need the builtins/past.builtins imports?

**Review by philipwinklerdelmic — APPROVED** [review]
Looks good.

File: src/odemis/cli/main.py [hunk 2]
```diff
@@ -331,6 +332,32 @@ def print_vattribute(name, va, pretty):
                 logging.info("Failed to convert %s to component names")
                 # Leave the value as-is
 
+        # Convert to nicer unit for user
+        if va.unit and va.unit == "rad" and isinstance(val, numbers.Real):
+            try:
+                val_converted = u" = %s°" % (math.degrees(val),)
```
Comment (Mahmood-B) [inline src/odemis/cli/main.py:338]: I'm wondering. Why actually do you make it unicode? Why not string?

File: src/odemis/cli/main.py [hunk 3]
```diff
@@ -282,7 +283,7 @@ def print_events(component, pretty):
         print_event(name, value, pretty)
 
 
-def print_vattribute(name, va, pretty):
+def print_vattribute(component, name, va, pretty):
```
Comment (Mahmood-B) [inline src/odemis/cli/main.py:286]: docstring?

File: src/odemis/cli/main.py [hunk 4]
```diff
@@ -331,6 +332,32 @@ def print_vattribute(name, va, pretty):
...
+        # For position, it's trickier, as the unit is on .axes
+        if (name == "position" and isinstance(va.value, dict) and
+            hasattr(component, "axes") and isinstance(component.axes, dict)
+           ):
+            pos_deg = {}
+            for an, pos in va.value.items():
+                try:
+                    axis_def = component.axes[an]
+                except KeyError:
+                    continue
```
Comment (Mahmood-B) [inline src/odemis/cli/main.py:354]: It's better to add a statement showing what axis missing: `logging.error("The axis %s is missing" %an)`. 
Such messages are not for the user, but for the developer if wants to change the code in the future. This was proven to me after adapting enzel code for meteor.

File: src/odemis/cli/main.py (hunk 1 above)
Comment (pieleric) [reply src/odemis/cli/main.py]: Not really, but to remove them, we need to update the code to Python 3-only, which I'm not going to do in this PR.
I'll put the lines back to where they were.

File: src/odemis/cli/main.py (hunk 2 above)
Comment (pieleric) [reply src/odemis/cli/main.py:345]: It's only for Python 2. In Python 3, all strings are unicode. We don't really need this anymore, as we now only support Python 3, but to follow the style of the rest of the code, I keep unicode here too. One day, someone will convert this code to Python 3 only, but that day is not today ;-)

File: src/odemis/cli/main.py (hunk 4 above)
Comment (pieleric) [reply src/odemis/cli/main.py:362]: Adding log statements affect the user, so if it's not for the user, I'd suggest to pass the information a different way. That said, I guess it's fine to tell the user that .position has axes not in .axes. I'll add some warnings.

**Review by Mahmood-B — APPROVED** [review]

**Pull request merged**